src/include/switch_types.h

```diff
@@ -1630,6 +1630,7 @@ typedef enum {
 	CF_VIDEO_READ_TAPPED,
 	CF_VIDEO_WRITE_TAPPED,
 	CF_DEVICES_CHANGED,
+    CF_LATE_MEDIA,
```

Comment:
maybe name this CF_LATE_OFFER ?

Reply:
@dragos-oancea  Thanks for review, have addressed review comment